examples/tutorials/Introduction_to_PROTACs.ipynb

```diff
@@ -0,0 +1,1968 @@
+{
```

Comment:
<p>Let's do scaffold split as well. Scaffold is generally a better test. Also let's visualize compounds in the train and test with scaffold split to see what these look like</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4000/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>